test/components/DayPickerRangeController_spec.jsx

```diff
@@ -796,6 +796,140 @@ describe('DayPickerRangeController', () => {
         });
       });
 
+      describe('blocked', () => {
+        describe.skip('focusedInput did not change', () => {
```

Comment:
This is one of the two tests I referenced in my commit message. I wasn't sure if it was appropriate to leave them out, or if it getting called 183 times actually isn't expected.